examples/server/public/index.html

```diff
@@ -220,7 +223,9 @@
       mirostat_tau: 5, // target entropy
       mirostat_eta: 0.1, // learning rate
       grammar: '',
-      n_probs: 0, // no completion_probabilities
+      n_probs: 0, // no completion_probabilities,
+      image_data: [],
```

Comment:
Saving image data with other params to `localStorage` (and loaind it from there) may be problematic when the user re-opens their browser. It still send it to the server, but not shown on the UI in this case. What about saving it separately to `sessionStorage`, or not saving at all?

Reply:
Sure, I'm not sure how to do this though. Feel free to update as necessary

Reply:
Fixed.